examples/ota-requestor-app/infineon/cyw30739/args.gni

```diff
@@ -17,6 +17,6 @@ import("${chip_root}/src/platform/Infineon/CYW30739/args.gni")
 
 cyw30739_sdk_target = get_label_info(":sdk", "label_no_toolchain")
 
-chip_openthread_ftd = true
+chip_openthread_ftd = false
```

Comment:
Curious why by default you are moving from FTD to MTD?

Reply:
Because the ota-requestor-app won't build unless changing it to MTD.
To support 5 fabrics, our platform needs an additional 16k flash space for the KVS.
But there is no space left in the ota-requestor-app, the only way I can do is to change it to MTD to reduce its code size.